### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by skovhus**
Fixes https://github.com/mads-hartmann/hatch/issues/18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ hatch is a tool that makes it easy to build products using AWS. It uses
 conventions to automate the creation of things like static website, HTTP API's
 etc. quick and easy.
 
+[![Build Status](https://travis-ci.org/skovhus/jest-codemods.svg?branch=master)](https://travis-ci.org/skovhus/jest-codemods)
```
Comment (mads-hartmann) [inline README.md]: I'd prefer if this pointed to the hatch build status 😉

Comment (skovhus) [inline README.md]: Haha, just checking if you were awake or too drunk on red wine!

Comment (mads-hartmann) [inline README.md]: They only served one glass at dinner, unfortunately 😂

**Comment by skovhus** [comment]
Review again :)

**Pull request merged**